e2e-tests/tfm/cypress/e2e/journeys/facility/facility-details.spec.js

```diff
@@ -0,0 +1,96 @@
+import relative from '../../relativeURL';
+import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import MOCK_DEAL_MIA from '../../../fixtures/deal-MIA';
+import MOCK_DEAL_MIN from '../../../fixtures/deal-MIN';
+import { T1_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../e2e-fixtures';
+
+context('Facility page', () => {
+  let ainDealId;
+  let miaDealId;
+  let minDealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+      ainDealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+      cy.createFacilities(ainDealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(ainDealId, dealType, T1_USER_1);
+    });
+
+    cy.insertOneDeal(MOCK_DEAL_MIA, BANK1_MAKER1).then((insertedDeal) => {
+      miaDealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_MIA;
+
+      cy.createFacilities(miaDealId, mockFacilities, BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(miaDealId, dealType, T1_USER_1);
+    });
+
+    // Create a new MIN deal based on the mock data
+    const newMinDeal = {
+      ...MOCK_DEAL_MIN,
+      _id: undefined, // Remove the _id so MongoDB will generate a new one
```

Comment:
I am not sure why this has been included?

Reply:
The error is quite clear we need to ensure the `_id` are unique when spreading `MOCK_DEAL_MIN` please follow the same approach as `MOCK_DEAL_AIN`.